curriculum/structure/superblocks/full-stack-developer.json

```diff
@@ -869,6 +869,11 @@
       "dashedName": "backend-javascript",
       "comingSoon": true,
       "modules": [
+        {
+          "dashedName": "introduction-to-nodejs",
+          "comingSoon": true,
+          "blocks": ["lecture-node-js-core-libraries"]
```

Comment:
this seems to be in the wrong module, looking at the worksheet, it should be in the nodejs-core-libraries module